index.html

```diff
@@ -0,0 +1,307 @@
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id="btn4">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- about section -->
+
+    <section class="about" id="about">
+        <div class="about_me">
+            <h2>About Myself</h2>
+            <p>
+                Hello I’m a software developer! I can help you
+                build a product , feature or website Look through
+                some of my work and experience! If you like what
+                you see and have a project you need coded, don’t 
+                hesitate to contact me.
+            </p>
+            <h3>LET'S CONNECT</h3>
+              <ul class="social">
+                <li>  <a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li>  <a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li>  <a href="#"><i class="fa-brands fa-mastodon"></i></a></li>
+                  <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                  <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+            <button>Get my resume</button>
+        </div>
+        <div class="expertise">
+            <div class="domaine">
+                <h2>languages</h2>
+                <i class="fa-solid fa-chevron-down"></i>
+            </div>
+            <div class="list_language">
+                <div class="lang"><img src="./images/js.png" alt="js"><h3>javaScript</h3></div>
+                <div class="lang"><img src="./images/html.png" alt="js"><h3>HTML</h3></div>
+                <div class="lang"><img src="./images/css.png" alt="js"><h3>CSS</h3></div>
+            </div>
+            <div class="domaine dom2">
+                <h2>Framework</h2>
+                    <i class="fa-solid fa-chevron-right"></i>
+            </div>
+            <div class="domaine dom3">
+                <h2>Skills</h2>
+                    <i class="fa-solid fa-chevron-right"></i>
+            </div>
+        </div>
+    </section>
+
+    <!-- contact section -->
+
+    <section class="contact scroll" id="contact">
+        <h2>Contact me</h2>
+        <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/mwkjpdrl" method="POST">
+            <input type="text" name="name" id="name" placeholder="Drissa" maxlength="30" class="user" required>
+            <input type="email" name="email" id="email" placeholder="dcopm@gmail.com" class="user" required>
+            <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] Let's add some padding inside the inputs to match the design. Currently, the text inside the inputs is sticking to the left. The `font-size` inside the textarea is looking a bit bigger when compared to the template. Kindly fix it accordingly. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220439591-d17b4012-dc17-4508-b095-bd6d217f8586.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220439585-0e90700a-7cec-4e88-a4be-f28cc2beb154.png" width="300">|

Reply:
ok